src/main/scala/com/box/castle/core/committer/states/CommittingBatch.scala

```diff
@@ -118,7 +123,17 @@ trait CommittingBatch extends CommitterActorBase
     // We commit the consumer offset here without waiting on the result, we are assuming it will succeed most of the time
     commitConsumerOffset(batch.nextOffset, metadata)
 
-    becomeFetchingData(OffsetAndMetadata(batch.nextOffset, metadata))
+    // If BatchSizeManager is enabled then goto Idling state with a specific non-zero delay
+    batchSizeManagerOption match {
+      case Some(batchSizeManager) =>
+        val delay = batchSizeManager.getDelay(committerActorId)
+        if(delay.getMillis > 0)
```

Comment:
does joda time have a typed 0 constant?  Can this be changed to something like
`if (delay > Duration.ZERO)` ?